src/components/AppNavigationToggle/AppNavigationToggle.vue

```diff
@@ -72,6 +83,8 @@ export default {
 	&:hover,
 	&:focus {
 		opacity: $opacity_full;
+		background-color: var(--color-background-hover);
+		border-radius: var(--border-radius-pill);
```

Comment:
I would argue that the radius should most likely be half the size of the element, and that this value have nothing to do here (as it's 100px and referring to another usage), but it works :grin:

Reply:
@skjnldsv @jancborchardt How about #1844?

Reply:
yes

Reply:
I will close this PR in favor of #1844 then.